indico/modules/events/editing/client/js/components/FileManager/reducer.js

```diff
@@ -0,0 +1,110 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import _ from 'lodash';
+import {combineReducers} from 'redux';
+import * as actions from './actions';
+
+export default combineReducers({
+  fileTypes: (state = [], action) => {
+    const fileType = _.find(state, {id: action.fileTypeId});
+    switch (action.type) {
+      case actions.MARK_UPLOADED:
```

Comment:
```suggestion
      case actions.MARK_UPLOADED: {
      ...
      }
```

Reply:
Just for the sake of style? Because the others are there for a reason.

Reply:
I think curly braces only make sense for cases if you use `let` or `const` and thus need the scope

Reply:
Not only, this will create another scope for all variables defined in that `case` (even though it might not be necessary in this case)

Reply:
I think it's better to leave them as they are, since there are no `let` or `const` there, for now.